config/initializers/mailjet.rb

```diff
@@ -2,5 +2,8 @@
 Mailjet.configure do |config|
   config.api_key = ENV['MAILJET_API_KEY']
   config.secret_key = ENV['MAILJET_API_SECRET']
-  config.default_from = 'noreply@covoiturage-libre.fr'
-end
\ No newline at end of file
+
+  mail_host = Rails.application.config.action_mailer.default_url_options[:host]
```

Comment:
same idea here, are we sure action_mailer will be configured with a host ? If none (because faulty ENV or another reason) we shouldn't try and use it below.

Reply:
Thanks I updated to ensure a default in `config/application.rb` and then by transition here when Mailjet is used.